django_mongodb_backend/utils.py

```diff
@@ -28,7 +30,19 @@ def check_django_compatability():
         )
 
 
-def parse_uri(uri, *, db_name=None, test=None):
+def get_auto_encryption_options(crypt_shared_lib_path=None):
+    key_vault_database_name = "encryption"
+    key_vault_collection_name = "__keyVault"
+    key_vault_namespace = f"{key_vault_database_name}.{key_vault_collection_name}"
+    kms_providers = {"local": {"key": os.urandom(96)}}
+    return AutoEncryptionOpts(
+        key_vault_namespace=key_vault_namespace,
+        kms_providers=kms_providers,
+        crypt_shared_lib_path=crypt_shared_lib_path,
+    )
```

Comment:
It's still obvious to me from the design doc to what extent it's appropriate for this library to provide helpers to generate `AutoEncryptionOpts`.